pkg/controller/json_bind.go

```diff
@@ -36,8 +36,8 @@ func BindJSON(w http.ResponseWriter, r *http.Request, data interface{}) error {
 		return fmt.Errorf("content-type is not application/json")
 	}
 
-	defer r.Body.Close()
```

Comment:
🛑 I think this is introducing a bug. We have to close the main body, not the limited reader.